<launch>
    <arg name="mission_file" default="default.json"/>
    <arg name='min_dis' default='0.1'/>

    <param name='/patrol/gps_topic' value = '/mavros/global_position/global'/>
    <param name='/patrol/imu_topic' value = '/mavros/imu/data'/>
    <param name="/patrol/mission_file" value="$(arg mission_file)"/>
    <param name="/patrol/min_dis_between_waypoints" value="$(arg diff)"/>
    <node pkg="autopilot_boson" type="record_waypoints_json.py" name="waypoints_recorder_node" output='screen' />

</launch>
